### PR TITLE
chore(structured-list): remove next folder

### LINKED ISSUE #12489: Storybook controls: Revisit Structured list

### PR DESCRIPTION
**Pull request by aledavila**
Closes #12489 

#### Changelog

Moves v11 files out of next folder

#### Testing / Reviewing

Make sure stories are working as intended

### CODE REVIEW

File: packages/react/src/components/StructuredList/index.js [hunk 1]
```diff
@@ -4,58 +4,50 @@
...
-} from './StructuredList';
-import { createComponentToggle } from '../../internal/ComponentToggle';
+// import {
+//   StructuredListWrapper as StructuredListWrapperNext,
+//   StructuredListHead as StructuredListHeadNext,
+//   StructuredListInput as StructuredListInputNext,
+//   StructuredListBody as StructuredListBodyNext,
+//   StructuredListRow as StructuredListRowNext,
+//   StructuredListCell as StructuredListCellNext,
+// } from './next/StructuredList';
```
Comment (tay1orjones) [inline packages/react/src/components/StructuredList/index.js]:
```suggestion
```

File: packages/react/src/components/StructuredList/index.js [hunk 2]
```diff
@@ -4,58 +4,50 @@
...
-export const StructuredListWrapper = createComponentToggle({
-  name: 'StructuredListWrapper',
-  next: StructuredListWrapperNext,
-  classic: StructuredListWrapperClassic,
-});
+// export const StructuredListWrapper = createComponentToggle({
+//   name: 'StructuredListWrapper',
+//   next: StructuredListWrapperNext,
+//   classic: StructuredListWrapperClassic,
+// });
```
Comment (tay1orjones) [inline packages/react/src/components/StructuredList/index.js]:
```suggestion
```

File: packages/react/src/components/StructuredList/index.js [hunk 3]
```diff
@@ -4,58 +4,50 @@
...
-export const StructuredListHead = createComponentToggle({
-  name: 'StructuredListHead',
-  next: StructuredListHeadNext,
-  classic: StructuredListHeadClassic,
-});
+// export const StructuredListHead = createComponentToggle({
+//   name: 'StructuredListHead',
+//   next: StructuredListHeadNext,
+//   classic: StructuredListHeadClassic,
+// });
```
Comment (tay1orjones) [inline packages/react/src/components/StructuredList/index.js]:
```suggestion
```

File: packages/react/src/components/StructuredList/index.js [hunk 4]
```diff
@@ -4,58 +4,50 @@
...
-export const StructuredListInput = createComponentToggle({
-  name: 'StructuredListInput',
-  next: StructuredListInputNext,
-  classic: StructuredListInputClassic,
-});
+// export const StructuredListInput = createComponentToggle({
+//   name: 'StructuredListInput',
+//   next: StructuredListInputNext,
+//   classic: StructuredListInputClassic,
+// });
```
Comment (tay1orjones) [inline packages/react/src/components/StructuredList/index.js]:
```suggestion
```

File: packages/react/src/components/StructuredList/index.js [hunk 5]
```diff
@@ -4,58 +4,50 @@
...
-export const StructuredListBody = createComponentToggle({
-  name: 'StructuredListBody',
-  next: StructuredListBodyNext,
-  classic: StructuredListBodyClassic,
-});
+// export const StructuredListBody = createComponentToggle({
+//   name: 'StructuredListBody',
+//   next: StructuredListBodyNext,
+//   classic: StructuredListBodyClassic,
+// });
```
Comment (tay1orjones) [inline packages/react/src/components/StructuredList/index.js]:
```suggestion
```

File: packages/react/src/components/StructuredList/index.js [hunk 6]
```diff
@@ -4,58 +4,50 @@
...
-export const StructuredListRow = createComponentToggle({
-  name: 'StructuredListRow',
-  next: StructuredListRowNext,
-  classic: StructuredListRowClassic,
-});
+// export const StructuredListRow = createComponentToggle({
+//   name: 'StructuredListRow',
+//   next: StructuredListRowNext,
+//   classic: StructuredListRowClassic,
+// });
```
Comment (tay1orjones) [inline packages/react/src/components/StructuredList/index.js]:
```suggestion
```

File: packages/react/src/components/StructuredList/index.js [hunk 7]
```diff
@@ -4,58 +4,50 @@
...
-export const StructuredListCell = createComponentToggle({
-  name: 'StructuredListCell',
-  next: StructuredListCellNext,
-  classic: StructuredListCellClassic,
-});
+// export const StructuredListCell = createComponentToggle({
+//   name: 'StructuredListCell',
+//   next: StructuredListCellNext,
+//   classic: StructuredListCellClassic,
+// });
```
Comment (tay1orjones) [inline packages/react/src/components/StructuredList/index.js]:
```suggestion
```

**Review by tay1orjones — COMMENTED** [review]
Just need to remove these

**Review by tay1orjones — APPROVED** [review]
🎉

**Review by tw15egan — APPROVED** [review]

**Pull request merged**